docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -52,20 +75,28 @@ The WordPress [wp-env package](https://www.npmjs.com/package/@wordpress/env) let
 
 The `wp-env` package requires Docker to be installed. There are instructions available for installing Docker on [Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/), [all other versions of Windows](https://docs.docker.com/toolbox/toolbox_install_windows/), [macOS](https://docs.docker.com/docker-for-mac/install/), and [Linux](https://docs.docker.com/v17.12/install/linux/docker-ce/ubuntu/#install-using-the-convenience-script).
 
-After confirming that the prerequisites are installed, install `wp-env` globally from the command-line using:
+After you have installed Docker, go ahead and install `wp-env` globally from the command-line using:
 
 ```sh
 npm -g install @wordpress/env
 ```
 
-This will be used in the next [Plugin step](wp-plugin.ms). To confirm it is installed and available, run:
+To confirm it is installed and available, run:
 
 ```sh
 wp-env --version
-> 1.4.0
+> 1.6.0
 ```
 
-### Alternatives
+The `wp-env` script is used to create a Docker WordPress environment, to start use:
+
+```sh
+wp-env start
```

Comment:
I'm not super familiar with this file, but it may be helpful to mention:
- configuring wp-env with the `.wp-env.json` file for project-specific settings
- If you're developing one plugin/theme primarily, you should run `wp-env start` from plugin/theme directory. (if you do so, it will mount the plugin/theme automatically)

If you want one environment for all your plugins/themes, you'd just have to create a `.wp-env.json` in some arbitrary location and add the plugins/themes to it. (then, you have to run `wp-env start` from the same directory as that config file

Reply:
Thanks, I've updated with this info. 👍 

This page is a generic page for getting a development environment setup, the intended audience is a plugin or theme developer. So this is right on point, I appreciate the additional info for using multiple plugins/themes.

Reply:
```
wp-env --version
> 1.6.0
```

The wp-env script is used to create a Docker WordPress environment, to use start from your plugin directory—if you are following the create block tutorial, this would be in the generated directory.
`wp-env start`

It is confusing going from working in terminal to suddenly mentioning the plugin directory. 
Where is the plugin directory? We need a step that goes from checking the wp-env to locating the plugin directory and then running `wp-env start`. This needs clarification.


"...if you are following the create block tutorial, this would be in the generated directory."
This seems out of context. As the context is setting up a local dev environment.